crates/fortitude_linter/src/rules/style/strings.rs

```diff
@@ -147,7 +153,9 @@ impl AstRule for AvoidableEscapedQuote {
         _settings: &CheckSettings,
         node: &'a Node,
         src: &'a SourceFile,
+        _symbol_table: &SymbolTables,
     ) -> Option<Vec<Diagnostic>> {
+        let _ = _symbol_table;
```

Comment:
I'm a bit confused by this line

Reply:
Ah, I think that's an lsp auto-fix to ignore the unused variable warning, I probably hit too many buttons and applied two different fixes.